src/frontend/screens/Audio/CreateRecording/index.tsx

```diff
@@ -1,40 +1,64 @@
 import React, {useEffect} from 'react';
-import {Text, View, StyleSheet} from 'react-native';
-import {WHITE} from '../../../lib/styles';
-
+import {BackHandler} from 'react-native';
 import {useNavigationFromRoot} from '../../../hooks/useNavigationWithTypes';
-import {CustomHeaderLeft} from '../../../sharedComponents/CustomHeaderLeft';
+import {RecordingActive} from './RecordingActive';
+import {RecordingDone} from './RecordingDone';
+import {RecordingIdle} from './RecordingIdle';
+import {useAudioRecording} from './useAudioRecording';
 
 export function CreateRecording() {
   const navigation = useNavigationFromRoot();
+  const {startRecording, stopRecording, reset, status, uri} =
+    useAudioRecording();
+
+  let currentState = 'idle';
+  if (status && status.isRecording) {
+    currentState = 'active';
+  } else if (uri) {
+    currentState = 'done';
+  }
```

Comment:
slight optimization would be: const currentState = status?.isRecording ? 'active' : uri ? 'done' : 'idle';